blocks/api/raw-handling/is-inline-content.js

```diff
@@ -15,6 +15,7 @@ export default function( HTML ) {
 
 function deepCheck( nodes ) {
 	return nodes.every( ( node ) => {
-		return isInline( node ) && deepCheck( Array.from( node.children ) );
+		return ( 'span' === node.nodeName.toLowerCase() || isInline( node ) ) &&
```

Comment:
If we're comparing a static string reference, why not just skip the `toLowerCase()`?

```js
return ( 'SPAN' === node.nodeName || isInline( node ) ) &&
```

Reply:
Yes, you are right, the improvement was applied.